src/applications/personalization/profile/components/connected-apps/ConnectedApp.jsx

```diff
@@ -33,7 +33,7 @@ export class ConnectedApp extends Component {
         <img
           className="va-connected-app-account-logo vads-u-margin-right--2p5"
           src={logo}
-          alt={`${title} logo`}
+          alt={``}
```

Comment:
Feels wrong to have an empty string for the `alt` attribute. I'm not sure the best way to deal with this, but maybe adding `aria-hidden="true"` and deleting the `alt` attribute entirely?

Reply:
eh I see Josh asked for this in particular. So 👍

Reply:
I agree with you. I just used the solution requested in the ticket. I just looked into your way, however our linting requires alt text to an image so if we did that we would also need to disable linting for that error and I kinda hate disabling the linter.